pkg/cmd/hgctl/installer/installer_k8s.go

```diff
@@ -35,6 +35,14 @@ type K8sInstaller struct {
 }
 
 func (o *K8sInstaller) Install() error {
+	// check if higress is installed by helm
```

Comment:
It would be better for us to first resolve the namespace from the configuration file and then check if higress is installed in the specific namespace.